libs/gl-client/src/node/mod.rs

```diff
@@ -1,5 +1,3 @@
-use std::time::Duration;
-
```

Comment:
Thanks for doing this cleanup. It may be better to split drive-by fixes like this into a separate commit, allowing us to cherry-pick them onto a bug-fix branch, and avoid conflicts if multiple people get annoyed by the lint warning and fix it (I did that in 3 branches so far, and rebase will just deduplicate the cherry-picked commits if we do it correctly).